pages/en/node-operators/transaction-failures.mdx

```diff
@@ -0,0 +1,53 @@
+import Page from "@reason/pages/Docs";
+import DocLink from "@reason/components/DocLink";
+export default Page ({ title: "Transaction failures" });
+
+# Transaction failures
+
+If a transaction fails to be applied, usually it would trigger 1 or more failures. There's 3 kind of transactions: payments, delegations, or zkApp transactions. There're several different kinds of failure in mina system. Some of the failures are specific to some kind of transactions. Let me explain them one by one below.
+
+## Source_not_present
+For payment, this means the mina node can not find the sender's account in the ledger.
+For delegation, this means the mina node can not find the delegator's account in the ledger.
+
+## Receiver_not_present
+This kind of failure only happens when a user tries to delegates to an non-exist account. If a user tries to send a payment to a new account, then this account would be created by this payment transaction.
+
+## Amount_insufficient_to_create_account
+For payment, this means the user tries to create a new account in this payment transaction, but the amount in the transaction is not enough for covering the account creation fee.
+For zkApp transaction, this means the smart contract tries to create 1 or more new accounts, but the amount in the smart contract is not enough for covering the account creation fee(s).
+
+## Source_insufficient_balance
+For payment, this means the sender does not have enough balance to cover the amount in the transaction.
+
+## Source_minimum_balance_violation
+For payment, this means the sender is a time-locked accounts and the unlocked funds is not enough to cover the amount in the transaction.
```

Comment:
Can we combine the explanations for payments and zkApps, since both can involve time-locked accounts?

How about:
"The unlocked funds in a time-locked account are insufficient to cover a transaction, either from a payment sender, or a transfer from an account in a zkApp"